e2e/cypress/tests/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -551,7 +551,7 @@ describe('Messaging', () => {
 
         // # Edit post by opening modal
         cy.getLastPostId().then((postId) => {
-            cy.clickPostDotMenu(postId);
+            cy.clickPostDotMenu(postId, 'RHS_ROOT');
```

Comment:
I've MM-T2140 still failed - https://automation-dashboard.vercel.app/cycles/510ab98e-f1bf-4ba6-985f-4c0a69b3d2a6
<img width="924" alt="Screen Shot 2022-07-20 at 6 26 47 AM" src="https://user-images.githubusercontent.com/5334504/179858958-9104893c-013a-44e5-ae5a-aab42155a9fc.png">

Reply:
For some reason the previous tests were affecting the failing one. I added a reload after view port change so that the center frame is loaded again properly before the test continues.